src/unix/fs.c

```diff
@@ -826,7 +826,8 @@ static int uv__fs_fstat(int fd, uv_stat_t *buf) {
 
 
 typedef ssize_t (*uv__fs_buf_iter_processor)(uv_fs_t* req);
-static ssize_t uv__fs_buf_iter(uv_fs_t* req, uv__fs_buf_iter_processor process) {
+static ssize_t uv__fs_buf_iter(uv_fs_t* req, uv__fs_buf_iter_processor process,
+                               int retry_on_eintr) {
```

Comment:
The reason why 923 exists is because we want to retry all fs operations on EINTR except close. I don't see that changing.